docs/en/observability/inventory-threshold-alert.asciidoc

```diff
@@ -35,16 +35,11 @@ You can also set advanced options such as the number of consecutive runs that mu
 In this example, Kubernetes Pods is the selected inventory type. The conditions state that you will receive
 a critical alert for any pods within the `ingress-nginx` namespace with a memory usage of 95% or above
 and a warning alert if memory usage is 90% or above.
+The chart shows the results of applying the rule to the last 20 seconds of data.
```

Comment:
I see in the screenshot that `FOR THE LAST` field is `1 second`, setting it to `1 minute` or even `5 minutes` would make more sense.
Also, maybe it is worth mentioning that the chart time range is 20 times the value of `FOR THE LAST` field which we call the look-back window.

<img src="https://github.com/elastic/observability-docs/assets/12370520/a95b1bcf-412f-4076-8dfe-e1ca836d7e2d" width=500 />